src/pybind/mgr/dashboard/tests/test_tools.py

```diff
@@ -178,3 +178,23 @@ def test_dict_contains_path(self):
         self.assertTrue(dict_contains_path(x, ['a']))
         self.assertFalse(dict_contains_path(x, ['a', 'c']))
         self.assertTrue(dict_contains_path(x, []))
+
+    def test_json_str_to_dict(self):
+        expected_result = {'a': 1, "b": "ñ"}
```

Comment:
Please prettify the code, you're mixing singe and double quotes.

Reply:
I agree.

Reply:
Done.